framework/test/jest/integration/specs/modules/chain/blocks/blocks/on_receive_block.spec.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+describe('integration test (blocks) - receiveBlockFromNetwork()', () => {
+	describe('receiveBlockFromNetwork (empty transactions)', () => {
+		describe('for valid block', () => {
+			it.todo('should add block to blockchain');
+		});
+
+		describe('forkThree', () => {
+			describe('validate block slot', () => {
+				describe('when generator is not a delegate', () => {
+					it.todo('should not add block to blockchain');
```

Comment:
Maybe move this to the describe and change the describe to an `it('when generator is not a delegate it should not add block to blockchain')`